db/database_test.go

```diff
@@ -3084,6 +3084,20 @@ func TestGetDatabaseCollectionWithUserDefaultCollection(t *testing.T) {
 
 }
 
+func TestClusterUUID(t *testing.T) {
+	bucket := base.GetTestBucket(t)
+	defer bucket.Close()
+
+	db, err := NewDatabaseContext(base.TestCtx(t), "db", bucket, false, DatabaseContextOptions{})
+	require.NoError(t, err)
+
+	if base.TestUseCouchbaseServer() {
+		require.Equal(t, 32, len(db.clusterUUID)) // no dashes in UUID
```

Comment:
this type of assertion will print out the actual string rather than something like `32 != 36` if it fails

```suggestion
		require.Len(t, db.clusterUUID, 32) // no dashes in UUID
```